src/codemodder/semgrep.py

```diff
@@ -31,29 +32,12 @@ def run_on_directory(yaml_files: List[Path], directory: Path):
         command.append(str(directory))
         joined_command = " ".join(command)
         logger.debug("Executing semgrep with: `%s`", joined_command)
+        # TODO: don't use the joined command
```

Comment:
because? I guess this could be a security risk

Reply:
I'm not sure about security it just seems cleaner since we already have a list and the API accepts a list.

Reply:
Actually in general it's going to be safer to pass a list with `shell=False` but the risk here was pretty low to begin with.